src/main/scala/io/citrine/lolo/transformers/Standardizer.scala

```diff
@@ -170,7 +203,7 @@ class StandardizerPrediction[T](
     */
   override def getUncertainty(includeNoise: Boolean = true): Option[Seq[Any]] = {
     baseResult.getUncertainty(includeNoise) match {
-      case Some(x) if outputTrans.isDefined => Some(x.map(_.asInstanceOf[Double] * rescale))
+      case Some(x) if outputTrans.isDefined => Some(x.map(_.asInstanceOf[Double] * outputRescale))
```

Comment:
I wonder if there's a straightforward way to do the type resolution on `x` instead of within the `map`.

Reply:
I think it's not possible because of type erasure

Reply:
Yeah, you will not be able to match on `Seq[Double]` becasue of erasure. You could with a `ClassTag` parameter I believe, but that would require changing the interface for all `getUncertainty` methods.